config_webrx.py

```diff
@@ -336,6 +336,36 @@
 # path to the aprs symbols repository (get it here: https://github.com/hessu/aprs-symbols)
 aprs_symbols_path = "/usr/share/aprs-symbols/png"
 
+# Uncomment the following to customize gateway beacon details reported to the aprs network
+#   Plese see Dire Wolf's documentation on PBEACON configuration for complete details:
+#   https://github.com/wb2osz/direwolf/raw/master/doc/User-Guide.pdf 
+
+# Symbol in its two-character form as specified by the APRS spec at http://www.aprs.org/symbols/symbols-new.txt
+#   Default: Receive only IGate (do not send msgs back to RF)  
+# aprs_igate_symbol = "R&"
+
+# Custom comment about igate 
+#   Default: OpenWebRX APRS gateway
+# aprs_igate_comment = "OpenWebRX APRS gateway"
+
+# Antenna Power, Height, Gain 
+#   Unspecified by default
+# Transmitter power (0 by default as only RX is supported)
+# aprs_igate_power = "0"
```

Comment:
there is no associated power on an OpenWebRX receiver, it can only receive. I don's see this changing anytime soon, so this setting shouldn't exist.

Reply:
Agree, included to offer user most customization options DireWolf supports in case they have a special use case.  

If preference is simplifying the config, I can remove from config file and leave it in an unlisted option or remove altogether.

Reply:
Removed from config file, but left in logic for now as an unlisted option.

Reply:
There is no use case that involves transmitting without impeding the receiver. Please remove!

Reply:
done.